File: PHP_implement/player.php
<?php

class Player
{
    var $name;
    var $age;
    var $job;
    var $salary;
    
    public function __construct($name, $age, $job, $salary) {
        $this->name = $name;
        $this->age = $age;
        $this->job = $job;
        $this->salary = $salary;
    }
}

$p1 = new Player('1','2','3','4');
print_r($p1);
?>

File: refactor idea in Java/Player.java
public class Player {
    public String name;
    public int age;
    public double salary;
    public String role;

    public Player() {
        name = "somebody";
        age = 10;
        salary = 0.1;
        role = "nobody";
    }

    public Player(String name, int age, double salary, String role) {
        this.name = name;
        this.age = age;
        this.salary = salary;
        this.role = role;
    }

}
